pixeltable/env.py

```diff
@@ -37,7 +36,7 @@ def get(cls) -> Env:
             cls._instance = Env()
         return cls._instance
 
-    def __init__(self):
+    def __init__(self) -> None:
```

Comment:
When does a type checker complain about the absence of -> None?

Reply:
Oh hmm, I think it should be ignoring this one. Maybe I put it in there casually rather than in response to a type checker message?

Ordinarily it ignores untyped methods unless you specify `--disallow-untyped-defs`.

Reply:
It does indeed complain about this. I don't know why. FWIW I prefer it with `-> None` but don't feel super strongly about it and there may be a way to disable that warning.